lib/mail/field.rb

```diff
@@ -173,6 +173,8 @@ def initialize(name, value = nil, charset = 'utf-8')
       @name = FIELD_NAME_MAP[@name.to_s.downcase] || @name
     end
 
+    attr_accessor :unparsed_value
```

Comment:
Should this be writable?

Reply:
I don't think so.  As I understood `Mail::Field` doesn't require origin value after parsing and exposes exclusively already parsed field value. If user needs to change value, she could use `#value=`, which re-creates field.

Reply:
Cool, so `attr_reader` is sufficient?

Reply:
I'm not sure I understood correctly "sufficient". I'd  make public only attribute getter.

Reply:
@Tensho `attr_accessor` creates getter and setter. Use `attr_reader`

Reply:
Ah, I see, sorry, I thought I placed `attr_reader` there :) my bad.